packages/versions/src/lib/getBuiltinVersions.ts

```diff
@@ -1,9 +1,14 @@
 import type { Versions } from './types';
 
+const FUEL_CORE = `0.41.7`;
 export function getBuiltinVersions(): Versions {
   return {
-    FUEL_CORE: '0.41.7',
+    FUEL_CORE: 'git:master',
     FORC: '0.66.7',
     FUELS: '0.99.0',
+    // @ts-expect-error - overriding until 0.42.0 is released
+    // because a lot of tests are failing
+    // given that the mismatch warning is appended to errors
+    FUEL_CORE,
```

Comment:
Won't the property `FUEL_CORE` returned by `getBuiltinVersions()` always carry the overwritten value (`0.41.7`)?

Reply:
What I meant is:

Why this?

```ts
const FUEL_CORE = `0.41.7`;
export function getBuiltinVersions(): Versions {
  return {
    FUEL_CORE: 'git:master',
    FORC: '0.66.7',
    FUELS: '0.99.0',
    // @ts-expect-error - overriding until 0.42.0 is released
    // because a lot of tests are failing
    // given that the mismatch warning is appended to errors
    FUEL_CORE,
  }
}
```

Instead of this?

```ts
export function getBuiltinVersions(): Versions {
  return {
    FUEL_CORE: '0.41.7', // TODO: change to 0.42 before merge
    FORC: '0.66.7',
    FUELS: '0.99.0',
  }
}
```

Reply:
Because the `FUEL_CORE` field gets automatically overwritten via [`rewriteVersions`](https://github.com/FuelLabs/fuels-ts/blob/ns/chore/fuel-core-0.42.0/packages/versions/scripts/rewriteVersions.ts#L69) in the [prebuild](https://github.com/FuelLabs/fuels-ts/blob/ns/chore/fuel-core-0.42.0/packages/versions/package.json#L38) script and gets set to `FUEL_CORE: 'git:master'`. The way I'm adding the new `FUEL_CORE` field doesn't conform to the regex in the script so it doesn't get overwritten.

Reply:
Certainly! That information has been purged from my swap! 😅